spec/controllers/users_controller_spec.rb

```diff
@@ -7,33 +7,22 @@
       expect(assigns(:user)).to be_instance_of User
     end
   end
+
   describe "POST create" do
     context "with valid user" do
       before { post :create, user: Fabricate.attributes_for(:user)}
-      it "sets @user" do
-        expect(assigns(:user)).to be_instance_of User
-      end
-      it "creates a valid user" do
-        expect(User.all.count).to eq(1)
-      end
-      it "redirects to sign_in path" do
-        response.should redirect_to sign_in_path
-      end
+
+      it("sets @user")                {expect(assigns(:user)).to be_instance_of User}
+      it("creates a valid user")      {expect(User.all.count).to eq(1)}
+      it("redirects to sign_in path") {expect_redirect_to_sign_in_path}
     end
 
-    context "without valid user" do
-      before do
-        post :create, user: {email: nil, password: "password", full_name: Faker::Name.name }
-      end
-      it "sets @user" do
-        expect(assigns(:user)).to be_instance_of User
-      end
-      it "does not create a new user" do
-        expect(User.all.count).to eq(0)
-      end
-      it "render the new template" do
-        response.should render_template :new
-      end
+    context "with invalid user" do
+      before { post :create, user: {email: nil, password: "password", full_name: Faker::Name.name }}
```

Comment:
You an can also use `Fabricate.attributes_for(:user, email: nil)` instead of the hash you are creating manually.